### ISSUE TITLE
try make a .d.ts

### ISSUE THREAD

**Issue by bluelovers** [CGamesPlay/node-callable-instance#1]
```ts
export declare const CallableInstance: ICallableInstance;
export interface ICallableInstance {
	new <T>(property: string | symbol): (...argv) => T;
	<T>(...argv: any[]): T;
}
```

**Comment by CGamesPlay** [comment]
Hey, thanks for contributing this! I don't use Typescript so I'm not completely sure how to use it. If you reopen this issue as a pull request I would be happy to merge!

**Comment by CGamesPlay** [comment]
closed by #2
